addons/l10n_ch/models/account_invoice.py

```diff
@@ -41,7 +41,8 @@ class AccountMove(models.Model):
     def _compute_l10n_ch_qr_is_valid(self):
         for move in self:
             move.l10n_ch_is_qr_valid = move.move_type == 'out_invoice' \
-                                       and move.partner_bank_id._eligible_for_qr_code('ch_qr', move.partner_id, move.currency_id, raises_error=False)
+                                       and move.partner_bank_id._eligible_for_qr_code('ch_qr', move.partner_id, move.currency_id, raises_error=False) \
+                                       and move.company_id.country_id.code == 'CH'
```

Comment:
Why wouldn't we display the QR code if we can? :thinking: 
It would be useful for the Swiss client, even if the company isn't

Reply:
Apparently this was discussed between the functional and the PO: 
> Bug confirmed by TSB and JOL: Swiss QR code should only be triggered if both bank accounts are Swiss IBAN bank account